lib/WPA/VersionedFlowSensitiveStat.cpp

```diff
@@ -46,8 +46,8 @@ void VersionedFlowSensitiveStat::performStat()
     versionStat();
     ptsSizeStat();
 
-    u32_t fiObjNumber = 0;
-    u32_t fsObjNumber = 0;
+    [[maybe_unused]] u32_t fiObjNumber = 0;
```

Comment:
Do not understand why this variable maybe unused. It is used at line 62.

Reply:
It's incremented there, but the actual value is never stored or used anywhere else. I'm compiling with clang trunk, which probably recently got a bit smarter about this kind of construct. Perhaps this used to be a statistic somewhere but is now never saved?

Reply:
I see. It would be good to store them in the stat map for later printing

Reply:
I created #822 to separate this out, if that's alright. I think that then has to be merged first (not sure how dependent PRs work in GitHub)